src/main/scripts/src/styles/variables.scss

```diff
@@ -1,18 +0,0 @@
-@import './template/variables';
-
-$primary-color: #f3f3f4;
-$secondary-color: #fff;
-
-$mobile-width: 768px;
-$logo-height: 75px;
-
-$shadow-default: 0 1px 5px 1px rgba(0, 0, 0, 0.2); // default shadow
-$shadow-up: 0 4px 8px 3px rgba(0, 0, 0, 0.11); // blurred shadow for 'higher' elements
-$shadow-down: 0 2px 2px 0 rgba(0,0,0,0.14), 0 3px 1px -2px rgba(0,0,0,0.2), 0 1px 5px 0 rgba(0,0,0,0.12); // strong down shadow
-$shadow-none: 0 0 0 0 rgba(0,0,0,0); // invisible shadow for transitions
-
-$post-it-width: 290px;
-
```

Comment:
This file is bootstrap unrelated. :D

Reply:
I know. These variables have been moved to `_lumeer-variables.scss` file while the Bootstrap-related variables have been moved to `_bootstrap-variables.scss` file.